models/keyword_test.go

```diff
@@ -193,4 +193,71 @@ var _ = Describe("Keyword", func() {
 			})
 		})
 	})
+
+	Describe("#GetKeyword", func() {
+		Context("given a keyword record in the database", func() {
+			It("returns a keyword record", func() {
+				user := FabricateUser(faker.Email(), faker.Password())
+				keyword := FabricateKeyword(faker.Word(), "https://www.google.com/search?lr=lang_en", user)
+				query := map[string]interface{}{
+					"id": keyword.Id,
+				}
+
+				keywordResult, err := models.GetKeyword(query)
+				if err != nil {
+					Fail(fmt.Sprintf("Get keyword failed: %v", err.Error()))
+				}
+
+				Expect(keyword.Id).To(Equal(keywordResult.Id))
+			})
+
+			Context("given a key is searched by a user", func() {
```

Comment:
Do you mean:

```suggestion
			Context("given the keyword belongs to the user", func() {
```

And below:

```suggestion
			Context("given the keyword does NOT belong to the user", func() {
```

Reply:
Updated https://github.com/Lahphim/go-crawler-challenge/pull/63/commits/31de8e0d39ef71c5815a64e606021db302854e17